tests/engine/test_custom_filters.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+
+from foxops.engine.rendering import create_template_environment, render_template_file
+
+
+async def test_rendering_custom_filter_increase_ip_add(tmp_path: Path):
```

Comment:
actually, testing these custom filters we should do without the templating around...

we can just write some tests that directly call these functions

Reply:
If we test directly the function we are not sure that these functions are available when we run the template. Apart from that we are calling functions in a different way (for example `ip_add_increase('1.1.1.1', 2)` vs `'1.1.1.1' | ip_add_increase(2)`). So I think it is a better test to check that is working inside a template.

Reply:
agreed!
Ideally - we should do both, verify that it's working in the template, but also that it covers all edge cases correctly.

I made some small adjustments to the tests - could you check out if that works for you? If yes, LGTM 🚀